package.json

```diff
@@ -48,6 +45,32 @@
     "typescript": "2.6.2",
     "typings-checker": "1.1.2"
   },
-  "tags": ["typescript", "express", "middleware", "fp-ts", "io-ts", "functional-programming"],
-  "keywords": ["typescript", "express", "middleware", "fp-ts", "io-ts", "functional-programming"]
+  "peerDependencies": {
+    "@types/cookies": "^0.7.1",
+    "@types/express": "4.11.0",
+    "@types/koa": "^2.0.43",
+    "@types/koa-router": "^7.0.27",
+    "cookies": "^0.7.1",
```

Comment:
`cookies`: is this dependency used anywhere?

Reply:
no, that can go now